src/GovernorCountingFractional.sol

```diff
@@ -53,6 +56,14 @@ abstract contract GovernorCountingFractional is Governor {
         return _proposalVotersWeightCast[proposalId][account] > 0;
     }
 
+    /**
+     * @dev Get the number of votes cast on proposal `proposalId` by account `account`.
+     * Useful if you intend to allow delegates to cast rolling, partial votes.
```

Comment:
Not a huge fan of how this is phrased. This extension *does* allow delegates to cast rolling, partial votes. I might rephrase to something like "Useful when creating an integration that allows delegates to cast rolling, partial votes."